mmv1/third_party/terraform/services/logging/resource_logging_project_bucket_config.go

```diff
@@ -282,6 +309,10 @@ func resourceLoggingProjectBucketConfigRead(d *schema.ResourceData, meta interfa
 		return fmt.Errorf("Error setting cmek_settings: %s", err)
 	}
 
+	if err := d.Set("index_configs", flattenIndexConfigs(indexConfigs.([]interface{}))); err != nil {
```

Comment:
```suggestion
	if err := d.Set("index_configs", flattenIndexConfigs(res["indexConfigs"].([]interface{}))); err != nil {
```

Reply:
My bad, was rushing to get this change in. Resolved now